Source/WebCore/platform/graphics/FontCascade.cpp

```diff
@@ -147,9 +147,6 @@ bool FontCascade::isCurrent(const FontSelector& fontSelector) const
 void FontCascade::updateFonts(Ref<FontCascadeFonts>&& fonts) const
 {
     m_fonts = WTFMove(fonts);
-    m_useBackslashAsYenSymbol = useBackslashAsYenSignForFamily(firstFamily());
```

Comment:
It's a bit surprising that this works. I guess it comes down to the fact that `RenderStyle::setFontDescription()` calls `FontCascade::FontCascade(FontCascadeDescription&& fd, float letterSpacing, float wordSpacing)` which ends up recomputing `useBackslashAsYenSignForFamily()`. And users have to call `RenderStyle::setFontDescription()` (instead of modifying it in-place) because `RenderStyle::fontDescription()` returns a const reference. So I guess this just works, which is pretty amazing.

Reply:
Yes, I think this patch is correct.